LocalFeeder/dss_functions.py

```diff
@@ -128,6 +147,7 @@ def get_capacitors(dss):
         datum["kVar"] = kvar
         datum["numPhases"] = NumPhase
         datum["power"] = dss.CktElement.Powers()[: 2 * NumPhase]
+        datum["node_names"] = get_all_nodes(buses[:1])  # second is 0
```

Comment:
Just checking - this is just the first node name right?

Reply:
Yes. Capacitors should follow this convention in all the cases I've seen so far.